functions/mobile-menu.js

```diff
@@ -0,0 +1,23 @@
+const hamburger = document.querySelector('.fa-bars');
+const mobileMenu = document.getElementById('mobile-menu');
+const cancelBtn = document.getElementById('btn-cancel');
+const displayMobile = Array.from(document.querySelectorAll('#mobile-menu ul li a'));
+
+function showMenu() {
+  hamburger.style.display = 'none';
+  mobileMenu.style.display = 'block';
+  document.body.style.overflow = 'hidden';
+}
+
+function hideMenu() {
+  document.body.style.overflow = 'hidden';
```

Comment:
- [ ] Good job with displaying the menu, but be careful, here, you are setting the body overflow to `hidden` after you hide your menu, which is making that you cannot scroll on the mobile version after clicking on any of the links you have, so please make sure to work on this issue.